README.md

```diff
@@ -0,0 +1,13 @@
+# Gonum quad  [![Build Status](https://travis-ci.org/gonum/quad.svg)](https://travis-ci.org/gonum/quad)  [![Coverage Status](https://img.shields.io/coveralls/gonum/quad.svg)](https://coveralls.io/r/gonum/quad)
+
+Quad provides numerical integration of single variable functions by quadrature for the Go language.
+
+## Issues
+
+If you find any bugs, feel free to file an issue on the github issue tracker. Discussions on API changes, added features, code review, or similar requests are preferred on the gonum-dev Google Group.
+
+https://groups.google.com/forum/#!forum/gonum-dev
```

Comment:
Oh, sorry for not checking first, my bad. Github markdown is smarter than I expected.